lib/bemhtml/runtime/index.js

```diff
@@ -389,6 +390,32 @@ BEMHTML.prototype.runOne = function runOne(json) {
   return res;
 };
 
+BEMHTML.prototype.tryRun = function tryRun(context, ent) {
+  var res;
+
+  try {
+    res = ent.run(context, ent);
+  } catch (e) {
+    console.error(e);
+
+    if (this.options.debug !== true)
```

Comment:
Break on exceptions is very useful at debugging.
But with this changes you can't use it.